pkg/helpers/helpers.go

```diff
@@ -125,6 +130,65 @@ func UpdateSubmarinerConfigStatusFn(cond metav1.Condition, managedClusterInfo co
 	}
 }
 
+type CRDClientHolder struct {
+	apiExtensionsClient apiextensionsclient.Interface
+}
+
+func NewCRDClientHolder() *CRDClientHolder {
```

Comment:
why we just use clientholder in library go directly

Reply:
`apiExtensionsClient` in `clientholder` is an internal, cannot call outside.